<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MachineDialog</class>
 <widget class="QDialog" name="MachineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>593</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="mainGridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="infoGroupBox">
     <property name="title">
      <string>Machine Information</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="summaryLabel">
        <property name="text">
         <string>Summary</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="summaryLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="iconLabel">
        <property name="text">
         <string>Icon</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="iconHorizontalLayout">
        <item>
         <widget class="QComboBox" name="iconComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="iconToolButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="document-open"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="configGroupBox">
     <property name="title">
      <string>Machine Configuration</string>
     </property>
     <layout class="QGridLayout" name="configGridLayout">
      <item row="1" column="0">
       <widget class="QLineEdit" name="configLineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="configToolButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset theme="document-open"/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="configGuideLabel">
        <property name="text">
         <string>Enter the config file used with the 86Box emulator. You can create a new one or import an existing one. The 86Box emulator adds new directories and files next to the config file, so it is a good idea to have a separate directory for each virtual machine.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="commandsGroupBox">
     <property name="title">
      <string>Custom Commands</string>
     </property>
     <layout class="QGridLayout" name="commandsGridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="commandsGuideLabel1">
        <property name="text">
         <string>The lines below can be used to define custom commands for the machine. You can leave them empty to use default commands from the preferences.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="startCommandLabel">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="startCommandLineEdit">
        <property name="placeholderText">
         <string>Using default command</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="settingsCommandLabel">
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="settingsCommandLineEdit">
        <property name="placeholderText">
         <string>Using default command</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="commandsGuideLabel2">
        <property name="text">
         <string>&lt;p&gt;Variables for commands:&lt;/p&gt;
&lt;ul&gt;
&lt;li&gt;&lt;b&gt;{86box}&lt;/b&gt; is replaced with the 86Box emulator executable&lt;/li&gt;
&lt;li&gt;&lt;b&gt;{config}&lt;/b&gt; is replaced with a configuration file for the selected virtual machine&lt;/li&gt;
&lt;/ul&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="commandsGuideLabel3">
        <property name="text">
         <string>The literal characters { and } may be included by preceding them with the same character. &quot;{{&quot; means &quot;{&quot; and &quot;}}&quot; means &quot;}&quot;.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="advancedPushButton">
     <property name="text">
      <string>Advanced</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>248</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
